Add remote working tips to communication activities (#886)
publiccodenet/about: activities/communication/communication-remote-first.md

---
type: Resource
---

> 

Reviewer: Add a top level heading (with one #) with the title of the page.

Author: Done!

As we aim to work remote-first, and a lot of our work with codebase communities happens remotely, here are some tips on communication in a remote-first environment.

> ## Questioning video-call as default. 

Reviewer 2: ```suggestion
## Questioning video-call as default
```

This is definitely down to individual preferences, but video calls may be more taxing on the eyes and require more points to pay attention to. 

> Some people on the team found that when having 1-on-1 check-in calls or chats, doing it over a phone call while walking around outside worked just as well (or better) and was less taxing/tiring. 

Reviewer 2: ```suggestion
Some people on the team found that when having 1-on-1 check-in calls or chats, doing it over a phone call while walking around outside worked just as well (or better) and was less tiring. 
```

> ## 'Closing the loop' on open-ended requests.

Reviewer 2: ```suggestion
## 'Closing the loop' on open-ended requests
```

> We identified a potentially harmful patterns around open requests not being 'closed'. 

Reviewer: ```suggestion
We identified a potentially harmful pattern around open requests not being 'closed'. 
```

So for example, someone sends an open ended request for input or help or review, but doesn't hear back from anyone.

> This can be demotivating for the initiator (feeling of shouting into the void, not being taken seriously, valued or respected, and feeling like doing meaningless work), expensive for the organisation (many people reading, thinking about it, closing, opening it later again, without coming to a conclusion), and a cycle of the item resurfacing without resolution.

Reviewer 2: ```suggestion
This can be demotivating for the initiator (feeling of shouting into the void, not being taken seriously, valued or respected, and feeling like doing meaningless work), expensive for the organisation (many people reading, thinking about it, closing, opening it later again, without coming to a conclusion), and lead to a cycle of the item resurfacing without resolution.
```

> While this loop is easily closed in a collocated working environment (reach over and say hey), a remote environment makes this more tricky.

Reviewer: ```suggestion
While this loop is easily closed in a co-located working environment (reach over and say hey), a remote environment makes this more tricky.
```

> We identified several ways to mitigate this by trying to get a culture around:

Reviewer 2: Suggest making this a bulleted, not numbered list, since these are unordered alternatives rather than sequential steps.

Reviewer 2: Also, this list uses 'you' for both the initiator and a respondent. Suggest either identifying roles better, or splitting into advice for initiators and advice for respondents.

Author: Done! Thanks for the helpful advice.

> 1) Don't deprioritise with silence, but actively say you can't/won't prioritise something

Reviewer: ```suggestion
1) Don't deprioritize with silence, but actively say you can't/won't prioritize something
```

2) Set a deadline for responses, and signal what you will do after that deadline (move ahead, or stop working on it if no other interest,...) 
3) Signal clearly if/when this is a blocker to your work that prevents you from moving on

> 4) Where possible ask individuals for input over an undefined group 

Reviewer: ```suggestion
4) Where possible, ask individuals for input over an undefined group 
```

5) Where possible, provide clear context, aims and expectations around the request

> 6) When you take on an open request, try signal whether it is something you will prioritise, or something you will do *if* you have time and space

Reviewer: ```suggestion
6) When you take on an open request, try signaling whether it is something you will prioritize, or something you will do *if* you have time and space
```

> 6) When you take on an open request, try signaling whether it is something you will prioritize, or something you will do *if* you have time and space

Reviewer 2: ```suggestion
6) When you take on an open request, try signaling whether it is something you will prioritize, or something you will only do *if* you have time and space
```
